source/doc-odm-user-guide/template-editor.rst

```diff
@@ -8,13 +8,13 @@ Accessing the Template Editor
 
 To get started, click **Set up templates** on the Dashboard:
 
-.. image:: images/set-up-templates.png
+.. image:: images/quickstart_user_dashboard.png
    :scale: 30 %
    :align: center
 
 Or use the shortcut dock and select **Template Editor**:
 
-.. image:: images/shortcuts-template-editor.png
+.. image:: images/shortcuts_1_37.png
```

Comment:
this image isn't shown, see here: https://odm-user-guide.readthedocs.io/en/changes_0/doc-odm-user-guide/template-editor.html

Reply:
Lena, did you check the rendered version? Other content looks good?